Vagrantfile

```diff
@@ -29,6 +29,8 @@ nightly_release = ENV['NIGHTLY_RELEASE'] || ''
 node_os = ENV['CONTIV_NODE_OS'] || 'centos'
 base_ip = ENV['CONTIV_IP_PREFIX'] || '192.168.2.'
 num_nodes = ENV['CONTIV_NODES'].to_i == 0 ? 3 : ENV['CONTIV_NODES'].to_i
+num_vm_cpus = ENV['CONTIV_CPUS'].to_i == 0 ? 4 : ENV['CONTIV_CPUS'].to_i
```

Comment:
Can you add this change for the k8s Vagrantfile in https://github.com/contiv/netplugin/blob/master/vagrant/k8s/Vagrantfile too, please?

Also, can you add the bug ID CNTV-88 in the PR description for tracking?

Reply:
Per discussion with @unclejack, I'll submit a separate PR to make similar changes for the k8s Vagrantfile in https://github.com/contiv/netplugin/blob/master/vagrant/k8s/Vagrantfile.